lib/fa2/nft/nft.impl.jsligo

```diff
@@ -0,0 +1,232 @@
+#import "../common/assertions.jsligo" "Assertions"
+
+#import "../common/errors.mligo" "Errors"
+
+#import "../common/tzip12.datatypes.jsligo" "TZIP12"
+
+#import "../common/tzip12.interfaces.jsligo" "TZIP12Interface"
+
+#import "../common/tzip16.datatypes.jsligo" "TZIP16"
+
+export namespace NFT implements TZIP12Interface.FA2{
+   export type ledger = big_map<nat, address>;
+   type operator = address;
+   export type operators = big_map<[address, operator], set<nat>>;
+   export type storage = {
+      ledger: ledger,
+      operators: operators,
+      token_metadata: TZIP12.tokenMetadata,
+      metadata: TZIP16.metadata
+   };
+   type ret = [list<operation>, storage];
+   //  operators 
+   /**
+* Check if the intented transfer is sent from the same sender as from field, otherwise check if the sender is part of the operator authorized to receive this token
+* @param operators :  operator bigmap
+* @param from_ : transfer from address
+* @param token_id : token_id to test  
+*/
+
+   export const assert_authorisation = (
```

Comment:
Same thing, it should be only in assertions.jsligo

Reply:
same remark